spynnaker/pyNN/models/spike_source/spike_source_poisson.py

```diff
@@ -532,6 +532,10 @@ def set_recording_spikes(self, new_state=True):
             self._spike_recorder.record != new_state)
         self._spike_recorder.record = new_state
 
+    @overrides(AbstractSpikeRecordable.get_spikes_sampling_interval)
+    def get_spikes_sampling_interval(self):
+        return globals_variables.get_simulator().machine_time_step
```

Comment:
As with the SpikeSourceArray...

Reply:
In this case NO.  As this class has stuff like generate_data_specification this is where the sampling interval belongs.

Reply:
OK, but then the data might be replicated - there is also a sampling interval in the recorder (unless there isn't)

Reply:
As above, this isn't yet critical, as long as it works in the next one ;)

Reply:
As above I am not sure sampling Interval belongs in recorder.

Keep trying to test that but on top of an unstable master this is proving difficult.

Reply:
Will look at this later when I do selective recording of spikes